src/standalone/Vegetation/Canopy.jl

```diff
@@ -545,6 +564,20 @@ function ClimaLSM.make_update_aux(
             )
         @. GPP = compute_GPP(An, K, LAI, Ω)
         @. gs = medlyn_conductance(g0, Drel, medlyn_factor, An, c_co2_air)
+        # update autotrophic respiration
+
+        Vcmax = compute_Vcmax(Vcmax25, T_air, To, R, ΔHVcmax)
+        @. Rd = dark_respiration(Vcmax25, β, f, ΔHRd, T_air, To, R)
```

Comment:
hmm we are currently computing Rd in two places now: https://github.com/CliMA/ClimaLSM.jl/blob/7f56c218715bf72a797ab2b6eb4c1f36b479e84b/src/standalone/Vegetation/photosynthesis.jl#L192C1-L192C57

The same is true for Vcmax.

Is there an order of operations that works so that we compute these only once?

Reply:
Also, Vcmax is a function of T_canopy and not T_air.

Reply:
@AlexisRenchon any thoughts on the computation of Rd? This happens in the photosynthesis computation already. maybe we could update Rd prior to executing this line
https://github.com/CliMA/ClimaLSM.jl/blob/1f2d6a8a2bfdcdcb6c091a626d9567cba71c3fcf/src/standalone/Vegetation/Canopy.jl#L536
and then pass it in as an argument to `compute_photosynthesis`, and then we can remove this line:
https://github.com/CliMA/ClimaLSM.jl/blob/1f2d6a8a2bfdcdcb6c091a626d9567cba71c3fcf/src/standalone/Vegetation/photosynthesis.jl#L190 inside that function

and compute Ra as you are already doing?

Reply:
might double check that we are passing T_canopy here as well instead of T_air?